Editor/Window/ManagedEC2Page.cs

```diff
@@ -161,6 +162,12 @@ private void UpdateGUI()
             {
                 _statusIndicator.Set(State.InProgress, textProvider.Get(Strings.ManagedEC2DeployStatusDeploying));
             }
+            else if (stackStatus == StackStatus.RollbackComplete)
+            {
+                var errorText = textProvider.GetError(ErrorCode.StackStatusInvalid);
+                _statusIndicator.Set(State.Failed, errorText);
+                _deployStatusBox.Show(StatusBox.StatusBoxType.Error,  errorText, null, string.Format(Urls.AwsCloudFormationTemplate, _stateManager.Region), Strings.LabelDeploymentCloudFormationConsole);
```

Comment:
It would be nice if this linked to the problematic stack instead of linking to the list of stacks